src/components/Steps/Steps.scss

```diff
@@ -58,7 +59,9 @@
       @include theme-prop(color, icon-color);
       width: 15.5px;
       height: 15.5px;
-
+      &.disabled {
```

Comment:
what is ths?

Reply:
Meirav asked to fix bug with arrow icon, which wasn't having a disabled color, when the button was disabled